airflow/plural/docs/kubernetes-executor.md

```diff
@@ -0,0 +1,47 @@
+## Configuring KubernetesExecutor
+
+The [KubernetesExecutor](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/executor/kubernetes.html) 
+in Apache Airflow is a valuable choice for specific use cases and scenarios. It offers several advantages that make it a 
+suitable execution engine for running your Airflow tasks.
+
+### edit values.yaml
+
+You'll then want to edit `airflow/helm/airflow/values.yaml` in your installation repo with something like:
+
+```yaml
+airflow:
```

Comment:
there's also an annoying `createPluralSecret: true` config variable needed here

Reply:
@michaeljguarino - just added it (I think). Can you confirm that I got it right?